src/ast/ast_validations.c

```diff
@@ -183,13 +183,20 @@ static AST_Validation _ValidateMultiHopTraversal
 	return AST_INVALID;
 }
 
-// Verify that MERGE doesn't redeclare bound relations, and that 
-// one reltype is specified for unbound relations (created).
+// Verify that MERGE doesn't redeclare bound relations, that one reltype is specified for unbound relations, 
+// and that there is not variable length relationship
```

Comment:
"and that the entity is not a variable length pattern"

Reply:
Thanks, change done in [0344746](https://github.com/RedisGraph/RedisGraph/pull/2759/commits/03447464b4327f59d61f0a9ab5cd986bd424ed9f)